examples/features/torch_batch_process/batch_inference/generate_embedding/README.md

```diff
@@ -0,0 +1,21 @@
+# Example: use torch_batch_process for embedding generation
+
+## Use case
+
+One common LLM use case is information retrieval. The first step of information retrieval is to generate embedding on
+documents and upload to a vector database. When a user query come in, the system would retrieve documents most relevant
+to the query from the vector database. These documents are often used to supplement the prompt provided to the LLM to 
+improve the quality of answer.
+
+## How does torch_batch_process API help?
+
+In this example, we use the `torch_batch_process` API to 
```

Comment:
nits: 
* `the system would retrieve` -> `the system would first compute embedding of the query, and then retrieve...`
* `API to` -> `API to:`
* `4 workers` -> `4 workers,`